rs/nns/governance/api/src/ic_nns_governance.pb.v1.rs

```diff
@@ -3042,6 +3042,12 @@ pub mod governance {
             Option<governance_cached_metrics::NeuronSubsetMetrics>,
         #[prost(message, optional, tag = "39")]
         pub public_neuron_subset_metrics: Option<governance_cached_metrics::NeuronSubsetMetrics>,
+        #[prost(message, optional, tag = "40")]
+        pub still_losing_voting_power_neuron_subset_metrics:
```

Comment:
would this be "declining_voting_power", i.e. it is in the process of moving downwards?